src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { ToDo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: ToDo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    data-id={todo.id}
+    className={cn('TodoInfo', {
+      'TodoInfo--completed': todo.completed === true,
+    })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```
let's move this check here, the purpose of the UserInfo component is to simply display user information without any logic or checks